src/ConnectedMachine/ConnectedMachine.Autorest/README.md

```diff
@@ -85,102 +85,104 @@ directive:
           }
         ]
  
-  - from: swagger-document
-    where: $.definitions.Machine.properties.properties
-    transform: >-
-      return {
-          "x-ms-client-flatten": true,
-          "$ref": "#/definitions/MachineProperties",
-          "description": "Hybrid Compute Machine properties"
-        }
+  # - from: swagger-document
```

Comment:
If it's not used anymore, please remove it.

Reply:
fixed